IslandLanding/IslandLanding/Views/WinPage.xaml

```diff
@@ -30,7 +40,7 @@
         <Label.FormattedText>
           <FormattedString>
             <Span Text="{Binding UserTag}"/>
-            <Span Text=",  You have successfully landed on the island!"/>
+            <Span Text=" ,You have successfully landed on the island!"/>
```

Comment:
@amiraelsayedbadwy  did you change this?

Reply:
yes

Reply:
Please make this ", you have ...

Reply:
done